README.md

```diff
@@ -365,8 +365,13 @@ TokenContract.events.Transfer({
 });
 ```
 
-## Deploy
-Starting from version **1.9.4** Parity includes support for running Wasm contracts. Wasm support isn't enabled by default and needs to be specified in the "chainspec" file. `wasmActivationTransition` param sets a block number Wasm support should be activated. This is a sample "development chain" spec with Wasm enabled (based on https://paritytech.github.io/wiki/Private-development-chain):
+## Run node and deploy conract
+Now it's time to deploy our Wasm contract on the blockchain. We can ether test in own [local development chain](#option-1-setup-and-run-development-node) or publish it on the [public Kovan network](#option-2-run-kovan-node).
+
+### Option 1: setup and run development node
+Starting from version **1.9.4** Parity includes support for running Wasm contracts.
```

Comment:
it's 1.9.5

Reply:
oh, it's kovan rules since 1.9.5
general wasm contracts execution dates back to 1.7

Reply:
and also, it's probably just put as a requirement somewhere, not as a history note